plugin/test/unit/helpers/draft.config.spec.ts

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+import { resolve, join } from 'path'
+
+import {
+  copySync,
+  ensureFileSync,
+  readdir,
+  readJSON,
+  writeJSON,
+} from 'fs-extra'
+import { v4 as uuidv4 } from 'uuid'
+
+import { createMetadataFileAndCopyDatastore } from '../../../src/helpers/config'
+
+jest.mock('fs-extra')
+jest.mock('uuid')
+
+// TODO: Need proper TS types here!
+const mockReadJSON = readJSON as jest.MockedFunction<any>
+const mockEnsureFileSync = ensureFileSync as jest.MockedFunction<any>
+const mockCopySync = copySync as jest.MockedFunction<any>
+const mockWriteJSON = writeJSON as jest.MockedFunction<any>
+const mockReaddir = readdir as jest.MockedFunction<any>
+const mockUuidv4 = uuidv4 as jest.MockedFunction<any>
+
+/* eslint-disable max-nested-callbacks */
+/* eslint-disable max-lines-per-function */
```

Comment:
Disabled some ESLint rules here. I personally like nested `describe` blocks for clarity and allowing setup/teardown for the blocks. I do appreciate it can be confusing.

I assume the `max-lines-per-function` is triggering from the large `describe` block with multiple tests in it.

Reply:
I think there's soemthing to be said for a different eslint config for tests.

Reply:
100% agree. Not suggesting we change them. I just disabled to get these working.